run_dir/static/js/running_notes.js

```diff
@@ -100,13 +100,25 @@ function load_running_notes(wait) {
   // Clear previously loaded notes, if so
   note_url = get_note_url();
   $("#running_notes_panels").empty();
+  // From agreements tab
+  $("#invoicing_notes").empty();
+  inv_notes = [];
   return $.getJSON(note_url, function(data) {
     if(Object.keys(data).length == 0 || typeof data === 'undefined'){
       $('#running_notes_panels').html('<div class="well">No running notes found.</div>');
     } else {
       $.each(data, function(date, note) {
         $('#running_notes_panels').append(make_running_note(date, note));
+        if(note['category'].includes('Invoicing')){
+          inv_notes.push(date, note);
+        }
       });
+      if(inv_notes.length === 0){
+        $('#invoicing_notes').html('<div class="well">No running notes found.</div>');
+      }
+      else {
+        $('#invoicing_notes').append(make_running_note(inv_notes[0], inv_notes[1], true));
```

Comment:
This will only display the latest invoicing note and not more. You need to use a loop